templates/semantic-ui/invenio_app_rdm/footer.html

```diff
@@ -25,7 +25,7 @@ <h2 class="ui small header">{{ _("About") }}</h2>
           </div>
           <div class="rdm-footer five wide tablet three wide computer eight wide mobile column">
             <h2 class="ui small header">{{ _("Help") }}</h2>
-            {% trans snow="https://cern.service-now.com/service-portal/function.do?name=CDS" %}
+            {% trans snow="https://cern.service-now.com/service-portal?id=service_element&name=CDS-Service" %}
```

Comment:
Above, can you change the URL and the text to:
`/privacy-notice`
`Privacy Notice`